plugins/woocommerce/src/Internal/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php

```diff
@@ -509,24 +509,9 @@ private function add_organization_group_blocks() {
 		);
 		$product_catalog_section->add_block(
 			array(
-				'id'         => 'product-catalog-search-visibility',
-				'blockName'  => 'woocommerce/product-catalog-visibility-field',
-				'order'      => 20,
-				'attributes' => array(
-					'label'      => __( 'Hide in product catalog', 'woocommerce' ),
-					'visibility' => 'search',
-				),
-			)
-		);
-		$product_catalog_section->add_block(
-			array(
-				'id'         => 'product-catalog-catalog-visibility',
-				'blockName'  => 'woocommerce/product-catalog-visibility-field',
-				'order'      => 30,
-				'attributes' => array(
-					'label'      => __( 'Hide from search results', 'woocommerce' ),
-					'visibility' => 'catalog',
-				),
+				'id'        => 'product-catalog-search-visibility',
+				'blockName' => 'woocommerce/product-catalog-visibility-fields',
+				'order'     => 20,
```

Comment:
I don't like how we combined this into a single block, I believe we should keep this as before and just have it render a single checkbox. See the comment in regarding the `CatalogVisibility` by removing the additional checkbox.